pkg/admin/auth/oauth2.go

```diff
@@ -243,7 +241,7 @@ func makeKeyring() (keyring.Keyring, error) {
 		ServiceName:              serviceName,
 		KeychainName:             keyChainName,
 		KeychainTrustApplication: true,
-		FileDir:                  filepath.Join(util.HomeDir(), "~/.config/pulsar", "credentials"),
+		FileDir:                  filepath.Join("~/.config/pulsar", "credentials"),
```

Comment:
Would it be better to use `filepath.Join(util.HomeDir(), ".config/pulsar", "credentials")`?

Reply:
@fantapsody sorry, I don't think so. Because FileDir in keyring pkg will auto resolve `~/` to the users' home dir. See https://github.com/99designs/keyring/blob/929f178361c0be06c50f0a82598cbfa1a965f40f/config.go#L29

BTW, we should deprecate `util.HomeDir()` in favor of golang builtin `os.UserHomeDir()` as util.HomeDir() doesn't handle error cases.